.github/scripts/generate_binary_build_matrix.py

```diff
@@ -159,6 +162,30 @@ def arch_type(arch_version: str) -> str:
 
 FULL_PYTHON_VERSIONS = ["3.8", "3.9", "3.10", "3.11"]
 
+def get_DEFAULT_TAG() -> str:
+    """
+    Returns the default tag used by the manylinux builder images.
+    """
+    return DEFAULT_TAG
+
+def get_CONDA_CONTAINER_IMAGES() -> Dict[str, str]:
+    """
+    Returns a dict of GPU architecture to container image name.
+    """
+    return CONDA_CONTAINER_IMAGES
+
+def get_LIBTORCH_CONTAINER_IMAGES() -> Dict[Tuple[str, str], str]:
+    """
+    Returns a dict of (GPU architecture, ABI) to container image name.
+    """
+    return LIBTORCH_CONTAINER_IMAGES
+
+def get_WHEEL_CONTAINER_IMAGES() -> Dict[str, str]:
+    """
+    Returns a dict of GPU architecture to container image name.
+    """
+    return WHEEL_CONTAINER_IMAGES
+
```

Comment:
To be frank, this all feels as an unneeded complexity. Can you please explain why this is a preferred way?
```suggestion
```